pom.xml

```diff
@@ -159,19 +158,18 @@ limitations under the License.
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.9</version>
+            <version>3.12.0</version>
```

Comment:
This can be replaced with the [commons lang3 api plugin](https://plugins.jenkins.io/commons-lang3-api/dependencies/)